pkg/issuer/venafi/client/venaficlient.go

```diff
@@ -93,7 +96,16 @@ func New(namespace string, secretsLister internalinformers.SecretLister, issuer
 		return nil, err
 	}
 
-	vcertClient, err := vcert.NewClient(cfg)
+	// Using `false` here ensures we do not immediately authenticate to the
+	// Venafi backend. Doing so invokes a call which force the API Key usage
+	// on the TPP side. This auth method has been removed since 22.4 of TPP.
+	// This reults results in an APIKey usage error.
+	// Reference code from vcert library which still refers to the APIKey.
+	// ref: https://github.com/Venafi/vcert/blob/master/pkg/venafi/tpp/connector.go#L137-L146
+	//
+	// cert-manager uses the VerifyCredentials function below after the client
+	// has been created.
```

Comment:
```suggestion
	// Using `false` here ensures we do not immediately authenticate to the
	// Venafi backend. Doing so invokes a call which forces the use of APIKey 
	// on the TPP side. This auth method has been removed since 22.4 of TPP.
	// This results in an APIKey usage error.
	// Reference code from vcert library which still refers to the APIKey.
	// ref: https://github.com/Venafi/vcert/blob/master/pkg/venafi/tpp/connector.go#L137-L146
	//
	// cert-manager uses the VerifyCredentials function below after the client
	// has been created.
```